### PR TITLE
Devplay new

### PR DESCRIPTION
**Pull request by DevT999**
tokenId type is changed based on lock branch

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -23,6 +23,6 @@ module.exports = {
     node: true,
   },
   rules: {
-    'simple-import-sort/imports': 'error',
```
Comment (peak3d) [inline .eslintrc.js]: I don't see any reason to remove this.
yarn format automatically sorts your imports.

File: package.json [hunk 2]
```diff
@@ -112,7 +112,7 @@
     "hardhat-deploy-ethers": "^0.3.0-beta.7",
     "isomorphic-fetch": "^3.0.0",
     "mocha": "^8.2.1",
-    "prettier": "^2.2.1",
+    "prettier": "2.2.1",
```
Comment (peak3d) [inline package.json]: Why?

**Review by peak3d — CHANGES_REQUESTED** [review]
Pls. don't change the logic of our build / code style

File: .eslintrc.js (hunk 1 above)
Comment (DevT999) [reply .eslintrc.js]: Done

File: package.json (hunk 2 above)
Comment (DevT999) [reply package.json]: Reverted

File: yarn.lock [hunk 3]
```diff
@@ -13472,16 +13472,16 @@ prettier-plugin-solidity@^1.0.0-beta.1:
     solidity-comments-extractor "^0.0.7"
     string-width "^4.2.2"
 
+prettier@2.2.1, prettier@^2.1.2, prettier@^2.2.1:
```
Comment (peak3d) [inline yarn.lock]: Pls. revert this change, too! thx!

**Review by peak3d — CHANGES_REQUESTED** [review]
Better :-) One small last thing (see comment) and we are done
